e2e_tests/utils/utils.go

```diff
@@ -97,15 +97,18 @@ func getRegionFromZone(zone string) string {
 // pickTestRegionForArtifactRegistry selects a random zone from the configured zones to pull osconfig-agent package from AR & selected-region
 func pickTestRegionForArtifactRegistry() string {
 	zones := config.Zones()
-	randomIndex := rand.Intn(len(zones))
 
-	currentIndex := 0
-	randomZone := ""
-	for zone := range zones {
-		if currentIndex == randomIndex {
-			randomZone = zone
-		}
+	if len(zones) == 0 {
+		// default region for tests
+		return "us-central1"
+	}
+
+	zoneKeys := make([]string, 0, len(zones))
```

Comment:
There's maps.Keys() function which does this but it is in experimental, so I think it's okay not to use it.

Reply:
Thanks for letting me know, I don't know it. So, I used the classic way to do this.